### PR TITLE
Prow: Adds crontab job for cluster cleanup

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: prow/test-infra-cleanup-cluster.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#############################################
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+if [ "${CI:-}" == 'bootstrap' ]; then
```
Comment (nclando) [inline prow/test-infra-cleanup-cluster.sh]: I think you can forego this if block here?

**Review by nlandolfi — CHANGES_REQUESTED** [review]

**Comment by nlandolfi** [comment]
@istio-testing test this

**Comment by sebastienvas** [comment]
PTAL

**Review by nlandolfi — APPROVED** [review]
LGTM

**Comment by yutongz** [comment]
@nlandolfi try use /lgtm to add label for mungegithub, thanks :)

**Comment by yutongz** [comment]
@sebastienvas Please refer to [pr template](https://raw.githubusercontent.com/kubernetes/kubernetes/master/.github/PULL_REQUEST_TEMPLATE.md) to add release note (We are not going to keep this feature in test-infra, just want you to dogfood it)

**Comment by nlandolfi** [comment]
@sebastienvas looks like merge conflicts

**Pull request merged**